airflow/providers/openlineage/conf.py

```diff
@@ -51,6 +51,11 @@ def disabled_operators() -> set[str]:
     return set(operator.strip() for operator in option.split(";") if operator.strip())
 
 
+@cache
+def selective_enable() -> bool:
+    return conf.getboolean("openlineage", "selective_enable", fallback=False)
```

Comment:
We can use _CONFIG_SECTION as in all other functions 😄

Reply:
True. Changed.